content/kubernetes-platform-services/artifact-storage/chartmuseum.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "ChartMuseum"
+description: "Chartmuseum is an artifact storage for Helm charts."
+---
+
+[Chartmuseum](https://github.com/kubernetes-helm/chartmuseum) is an artifact storage
+for [Helm]({{< relref "tools/helm.md" >}}) charts.
+
+# Dependencies
+
+* [Kube2IAM]({{< relref "kubernetes-backing-services/iam/kube2iam.md" >}})
+* [External DNS]({{< relref "kubernetes-backing-services/external-dns/external-dns.md" >}})
+* [Kube Lego]({{< relref "kubernetes-backing-services/tls-management/kube-lego-lets-encrypt.md" >}})
+
+# Installation
+
+## Provision S3 Bucket and IAM Role
+
+Create a file in `/conf/kops-aws-platform/chart-repo.tf` with the following content
+
+{{% include-github title="Chartmuseum S3 bucket and IAM role" type="code-block" org="cloudposse" repo="terraform-root-modules" ref="0.1.5" file="/aws/kops-aws-platform/chart-repo.tf" language="hcl" %}}
+
+## Rebuild the Geodesic Module
+
+[Rebuild](/geodesic/module/) the module
+```shell
+> make build
```

Comment:
should it be `make docker/build`?

Reply:
Yea, let's refer to the one provided by the `build-harness` rather than rely on it being aliases in the `Makefile`